my-react-app/src/App.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState, useRef } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+
+
+//type component
+
+
+function AddButton(props: { class: string; })
+{
+  const bodyRef = useRef(document.body);
+  
+  const handleColor = (color: string) => {
+    console.log(color);
+    bodyRef.current.className = color;
+  }
+  return(
+    <>
+    <button onClick={ () => handleColor(props.class)} className={props.class} id="myButton"> ClassName : {props.class.substring(3)} </button>
```

Comment:
niemals ein css style mit hifle von ids setzen dazu bitte immer css klassen anlegen

da wir aber das plugin tailwind css installiert haben solltest du in deisem projekt nichts mit css lösen sondern immer nur tailwind klassen anwenden 

bitte dazu mal kurz ein youtube video anschauen was tailwind ist und wie man es benützt 

dann brauchst du auch kein css mehr schreiben in diesem projekt